Sources/macro.js

```diff
@@ -451,6 +451,12 @@ export function algo(publicAPI, model, numberOfInputs, numberOfOutputs) {
     model.inputConnection[port] = outputPort;
   }
 
+  function getInputConnection(port = 0) {
+    if (model.inputConnection[port]) {
+      return model.inputConnection[port];
+    }
+  }
```

Comment:
Should you return something no matter what?
I would not even try to guard it at that point unless you want a `null` instead of an `undefined`